src/lib/logger.js

```diff
@@ -14,8 +14,7 @@ export default class Logger {
       Vue.config.errorHandler = (error, vm, info) => {
         console.error(info);
         console.error(error);
-        Object.assign(error, { info, type: 'vue-error' });
-        Logger.write(error);
+        Logger.write({ message: error.toString(), info, type: 'vue-error' });
```

Comment:
Ok, I got how it works:
https://github.com/aeternity/superhero-wallet/blob/a43d7bec1e14801b9cf0e8741e1d195e5893c2ff/src/lib/logger.js#L73